daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dfdlx.xsd

```diff
@@ -123,106 +123,10 @@
 
   <xs:attributeGroup name="ExtLayeringAGQualified">
     <xs:attribute form="qualified" name="layerTransform" type="dfdlx:LayerTransformType" />
```

Comment:
Should we rename layerTransform to just layer now, since it is the only property left in the layer system?